manubot/ai_revision/ai_revision_command.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+
+def cli_process(args):
+    # set paths for content
+    content_dir = args.content_directory
+    if not content_dir.is_dir():
+        logging.warning(f"content directory does not exist: {content_dir}")
```

Comment:
Does continuing make sense if there's no content directory?

```suggestion
        raise SystemExit(f"content directory is not a directory or does not exist: {content_dir}")
```